### PR TITLE
[CBRD-24478] redefine spec

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CBRD-24478

* Change Spec

> * Remove deduplicate_min_keys
>* Unify deduplicate_fk_level to deduplicate_key_level
> * Change default value
>> - [ ] deduplicate_key_level=-1
>> - [ ] print_index_detail=no
> * Add parameters to cubrid.conf file for distribution
> * Set max value of level to 14 and use only short type
> * An index with level=0 does not indicate level in the with clause.
     However, in unloaddb, it is displayed by default.
> * Change the syntax to specify the level when creating FK
> * Add --no-with-option-print parameter to unloaddb tool.
> * <del>The is_deduplicate and deduplicate_key_level columns are added to the db_index view. </del>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ test_defaults: &test_defaults
           mv -f CUBRID/log /tmp/logs/cubrid_log
           mv -f cubrid-testtools/CTP/sql/log /tmp/logs/ctp_log
           find /home -name "core.*" | xargs -I{} mv -f {} /tmp/logs
-          #dmesg > /tmp/logs/dmesg.out
+          #dmesg > /tmp/logs/dmesg.out
```
Comment (hgryoo) [inline .circleci/config.yml]:
```suggestion
          #dmesg > /tmp/logs/dmesg.out
```

File: msg/en_US.utf8/utils.msg [hunk 2]
```diff
@@ -560,6 +560,7 @@ valid options:\n\
   -C, --CS-mode               client-server mode execution\n\
       --datafile-per-class    create a object file for each class; default: disabled\n\
       --split-schema-files    Split schema information by object and generate each file; Default: generate one schema file with all objects\n\
+      --no-with-option-print  Prints the specified WITH clause options when creating the index; default: disabled.\n\
```
Comment (hyunikn) [inline msg/en_US.utf8/utils.msg]:
```suggestion
      --no-with-option-print  Do not print the specified WITH clause options when creating the index.\n\
```

File: src/base/system_parameter.c [hunk 3]
```diff
@@ -729,13 +729,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
...
 #define PRM_NAME_STATDUMP_FORCE_ADD_INT_MAX "statdump_force_add_int_max"
 
-#ifndef NDEBUG
-#define PRM_NAME_USE_DEDUPLICATE_KEY_MODE_OID_TEST  "use_deduplicate_key_mode_oid_test"
-#endif
-#define PRM_NAME_DEDUPLICATE_MIN_KEYS      "deduplicate_min_keys"
-#define PRM_NAME_DEDUPLICATE_FK_LEVEL      "deduplicate_fk_level"
 #define PRM_NAME_DEDUPLICATE_KEY_LEVEL     "deduplicate_key_level"
-#define PRM_NAME_DEDUPLICATE_PRINT_LEVEL   "deduplicate_print_level"
+#define PRM_NAME_WITH_OPTION_PRINT         "with_option_print"
```
Comment (hyunikn) [inline src/base/system_parameter.c]:
```suggestion
#define PRM_NAME_PRINT_WITH_CLAUSE         "print_with_clause"
```

File: .circleci/config.yml (hunk 1 above)
Comment (ctshim) [reply .circleci/config.yml]: Thank you.

File: conf/cubrid.conf [hunk 4]
```diff
@@ -73,3 +73,9 @@ log_volume_size=512M
 # log_max_archives=2147483647
 log_max_archives=0 
 
+# for deduplicate (0~14)
+deduplicate_key_level=0
+with_option_print=no
+
+
+
```
Comment (hgryoo) [inline conf/cubrid.conf]: redundant newline
```suggestion
```

File: msg/en_US.utf8/utils.msg (hunk 2 above)
Comment (ctshim) [reply msg/en_US.utf8/utils.msg]: OK,  Thank you.

File: src/base/system_parameter.c (hunk 3 above)
Comment (ctshim) [reply src/base/system_parameter.c]: We will change each as follows.
print_index_detail, --skip-index-detail

thanks a lot.

**Review by hyunikn — APPROVED** [review]

**Comment by beyondykk9** [comment]
Please, merge wih develop to pass the sql test cases.

**Review by hgryoo — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Pull request merged**